[2023-06-02 17:44:05] [INFO ] org.springframework.boot.StartupInfoLogger.logStarting(StartupInfoLogger.java:55) com.sansec.fabrictest.FabricTestApplication - Starting FabricTestApplication using Java 1.8.0_241 on lican with PID 15668 (D:\SansecProject2023\fabric-test\target\classes started by lican in D:\SansecProject2023\fabric-test)
[2023-06-02 17:44:05] [INFO ] org.springframework.boot.SpringApplication.logStartupProfileInfo(SpringApplication.java:631) com.sansec.fabrictest.FabricTestApplication - No active profile set, falling back to 1 default profile: "default"
[2023-06-02 17:44:09] [INFO ] org.springframework.boot.web.embedded.tomcat.TomcatWebServer.initialize(TomcatWebServer.java:108) org.springframework.boot.web.embedded.tomcat.TomcatWebServer - Tomcat initialized with port(s): 8080 (http)
[2023-06-02 17:44:09] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.AprLifecycleListener - Loaded Apache Tomcat Native library [1.2.30] using APR version [1.7.0].
[2023-06-02 17:44:09] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.AprLifecycleListener - APR capabilities: IPv6 [true], sendfile [true], accept filters [false], random [true], UDS [true].
[2023-06-02 17:44:09] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.AprLifecycleListener - APR/OpenSSL configuration: useAprConnector [false], useOpenSSL [true]
[2023-06-02 17:44:09] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.AprLifecycleListener - OpenSSL successfully initialized [OpenSSL 1.1.1k  25 Mar 2021]
[2023-06-02 17:44:09] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.coyote.http11.Http11NioProtocol - Initializing ProtocolHandler ["http-nio-8080"]
[2023-06-02 17:44:09] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.StandardService - Starting service [Tomcat]
[2023-06-02 17:44:09] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.StandardEngine - Starting Servlet engine: [Apache Tomcat/9.0.75]
[2023-06-02 17:44:10] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/] - Initializing Spring embedded WebApplicationContext
[2023-06-02 17:44:10] [INFO ] org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.prepareWebApplicationContext(ServletWebServerApplicationContext.java:292) org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 4321 ms
[2023-06-02 17:44:10] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.coyote.http11.Http11NioProtocol - Starting ProtocolHandler ["http-nio-8080"]
[2023-06-02 17:44:10] [INFO ] org.springframework.boot.web.embedded.tomcat.TomcatWebServer.start(TomcatWebServer.java:220) org.springframework.boot.web.embedded.tomcat.TomcatWebServer - Tomcat started on port(s): 8080 (http) with context path ''
[2023-06-02 17:44:10] [INFO ] org.springframework.boot.StartupInfoLogger.logStarted(StartupInfoLogger.java:61) com.sansec.fabrictest.FabricTestApplication - Started FabricTestApplication in 7.948 seconds (JVM running for 11.218)
[2023-06-02 17:47:15] [ERROR] org.springframework.boot.SpringApplication.reportFailure(SpringApplication.java:821) org.springframework.boot.SpringApplication - Application run failed
java.lang.IllegalStateException: java.lang.IllegalStateException: Could not initialize Log4J2 logging from log4j2.xml
	at org.springframework.boot.context.logging.LoggingApplicationListener.initializeSystem(LoggingApplicationListener.java:344) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.context.logging.LoggingApplicationListener.initialize(LoggingApplicationListener.java:298) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.context.logging.LoggingApplicationListener.onApplicationEnvironmentPreparedEvent(LoggingApplicationListener.java:246) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.context.logging.LoggingApplicationListener.onApplicationEvent(LoggingApplicationListener.java:223) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:176) ~[spring-context-5.3.27.jar:5.3.27]
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:169) ~[spring-context-5.3.27.jar:5.3.27]
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:143) ~[spring-context-5.3.27.jar:5.3.27]
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:131) ~[spring-context-5.3.27.jar:5.3.27]
	at org.springframework.boot.context.event.EventPublishingRunListener.environmentPrepared(EventPublishingRunListener.java:85) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.SpringApplicationRunListeners.lambda$environmentPrepared$2(SpringApplicationRunListeners.java:66) ~[spring-boot-2.7.12.jar:2.7.12]
	at java.util.ArrayList.forEach(ArrayList.java:1257) ~[?:1.8.0_241]
	at org.springframework.boot.SpringApplicationRunListeners.doWithListeners(SpringApplicationRunListeners.java:120) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.SpringApplicationRunListeners.doWithListeners(SpringApplicationRunListeners.java:114) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.SpringApplicationRunListeners.environmentPrepared(SpringApplicationRunListeners.java:65) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.SpringApplication.prepareEnvironment(SpringApplication.java:343) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:301) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1303) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1292) ~[spring-boot-2.7.12.jar:2.7.12]
	at com.sansec.fabrictest.FabricTestApplication.main(FabricTestApplication.java:10) ~[classes/:?]
Caused by: java.lang.IllegalStateException: Could not initialize Log4J2 logging from log4j2.xml
	at org.springframework.boot.logging.log4j2.Log4J2LoggingSystem.loadConfiguration(Log4J2LoggingSystem.java:230) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.logging.log4j2.Log4J2LoggingSystem.loadConfiguration(Log4J2LoggingSystem.java:193) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.logging.AbstractLoggingSystem.initializeWithSpecificConfig(AbstractLoggingSystem.java:66) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.logging.AbstractLoggingSystem.initialize(AbstractLoggingSystem.java:57) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.logging.log4j2.Log4J2LoggingSystem.initialize(Log4J2LoggingSystem.java:169) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.context.logging.LoggingApplicationListener.initializeSystem(LoggingApplicationListener.java:332) ~[spring-boot-2.7.12.jar:2.7.12]
	... 18 more
Caused by: java.io.FileNotFoundException: D:\SansecProject2023\fabric-test\log4j2.xml (系统找不到指定的文件。)
	at java.io.FileInputStream.open0(Native Method) ~[?:1.8.0_241]
	at java.io.FileInputStream.open(FileInputStream.java:195) ~[?:1.8.0_241]
	at java.io.FileInputStream.<init>(FileInputStream.java:138) ~[?:1.8.0_241]
	at java.io.FileInputStream.<init>(FileInputStream.java:93) ~[?:1.8.0_241]
	at sun.net.www.protocol.file.FileURLConnection.connect(FileURLConnection.java:90) ~[?:1.8.0_241]
	at sun.net.www.protocol.file.FileURLConnection.getInputStream(FileURLConnection.java:188) ~[?:1.8.0_241]
	at java.net.URL.openStream(URL.java:1067) ~[?:1.8.0_241]
	at org.springframework.boot.logging.log4j2.Log4J2LoggingSystem.getConfigurationSource(Log4J2LoggingSystem.java:241) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.logging.log4j2.Log4J2LoggingSystem.load(Log4J2LoggingSystem.java:236) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.logging.log4j2.Log4J2LoggingSystem.loadConfiguration(Log4J2LoggingSystem.java:221) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.logging.log4j2.Log4J2LoggingSystem.loadConfiguration(Log4J2LoggingSystem.java:193) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.logging.AbstractLoggingSystem.initializeWithSpecificConfig(AbstractLoggingSystem.java:66) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.logging.AbstractLoggingSystem.initialize(AbstractLoggingSystem.java:57) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.logging.log4j2.Log4J2LoggingSystem.initialize(Log4J2LoggingSystem.java:169) ~[spring-boot-2.7.12.jar:2.7.12]
	at org.springframework.boot.context.logging.LoggingApplicationListener.initializeSystem(LoggingApplicationListener.java:332) ~[spring-boot-2.7.12.jar:2.7.12]
	... 18 more
[2023-06-02 17:47:32] [INFO ] org.springframework.boot.StartupInfoLogger.logStarting(StartupInfoLogger.java:55) com.sansec.fabrictest.FabricTestApplication - Starting FabricTestApplication using Java 1.8.0_241 on lican with PID 10860 (D:\SansecProject2023\fabric-test\target\classes started by lican in D:\SansecProject2023\fabric-test)
[2023-06-02 17:47:32] [INFO ] org.springframework.boot.SpringApplication.logStartupProfileInfo(SpringApplication.java:631) com.sansec.fabrictest.FabricTestApplication - No active profile set, falling back to 1 default profile: "default"
[2023-06-02 17:47:33] [INFO ] org.springframework.boot.web.embedded.tomcat.TomcatWebServer.initialize(TomcatWebServer.java:108) org.springframework.boot.web.embedded.tomcat.TomcatWebServer - Tomcat initialized with port(s): 8080 (http)
[2023-06-02 17:47:33] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.AprLifecycleListener - Loaded Apache Tomcat Native library [1.2.30] using APR version [1.7.0].
[2023-06-02 17:47:33] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.AprLifecycleListener - APR capabilities: IPv6 [true], sendfile [true], accept filters [false], random [true], UDS [true].
[2023-06-02 17:47:33] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.AprLifecycleListener - APR/OpenSSL configuration: useAprConnector [false], useOpenSSL [true]
[2023-06-02 17:47:33] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.AprLifecycleListener - OpenSSL successfully initialized [OpenSSL 1.1.1k  25 Mar 2021]
[2023-06-02 17:47:33] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.coyote.http11.Http11NioProtocol - Initializing ProtocolHandler ["http-nio-8080"]
[2023-06-02 17:47:33] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.StandardService - Starting service [Tomcat]
[2023-06-02 17:47:33] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.StandardEngine - Starting Servlet engine: [Apache Tomcat/9.0.75]
[2023-06-02 17:47:34] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/] - Initializing Spring embedded WebApplicationContext
[2023-06-02 17:47:34] [INFO ] org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.prepareWebApplicationContext(ServletWebServerApplicationContext.java:292) org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 1788 ms
[2023-06-02 17:47:34] [INFO ] org.apache.juli.logging.DirectJDKLog.log(DirectJDKLog.java:173) org.apache.coyote.http11.Http11NioProtocol - Starting ProtocolHandler ["http-nio-8080"]
[2023-06-02 17:47:34] [INFO ] org.springframework.boot.web.embedded.tomcat.TomcatWebServer.start(TomcatWebServer.java:220) org.springframework.boot.web.embedded.tomcat.TomcatWebServer - Tomcat started on port(s): 8080 (http) with context path ''
[2023-06-02 17:47:34] [INFO ] org.springframework.boot.StartupInfoLogger.logStarted(StartupInfoLogger.java:61) com.sansec.fabrictest.FabricTestApplication - Started FabricTestApplication in 2.98 seconds (JVM running for 3.972)
[2023-06-02 18:42:31] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-02 18:52:12] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-02 18:52:12] [INFO ] org.hyperledger.fabric.sdk.Channel.joinPeer(Channel.java:772) org.hyperledger.fabric.sdk.Channel - Channel{id: 1, name: test} joining Peer{ id: 3, name: peer0.org1.example.com, channelName: null, url: grpcs://peer0.org1.example.com:7051}.
[2023-06-02 18:52:16] [INFO ] org.hyperledger.fabric.sdk.Channel.joinPeer(Channel.java:804) org.hyperledger.fabric.sdk.Channel - Peer Peer{ id: 3, name: peer0.org1.example.com, channelName: test, url: grpcs://peer0.org1.example.com:7051} joined into channel Channel{id: 1, name: test}
[2023-06-02 18:52:16] [INFO ] org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1149) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 1, name: test} eventThread started shutdown: false  thread: null 
[2023-06-02 19:19:14] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-02 19:19:15] [INFO ] org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder.createNetModeTransaction(InstallProposalBuilder.java:244) org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder - Installing 'basicinfo::basicinfo::1.0' language Go chaincode from directory: 'D:\SansecData5\chaincode\src\basicinfo' with source location: 'src\basicinfo'. chaincodePath:'basicinfo'
[2023-06-02 19:19:20] [ERROR] org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4140) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 3, name: } Sending proposal with transaction: 24888c3eec3a4692b4fe30461eb1cab9d507b2b61f0a90bc7c5f8e0310364b6e to Peer{ id: 1, name: peer0.org1.example.com, channelName: null, url: grpcs://peer0.org1.example.com:7051} failed because of: gRPC failure=Status{code=UNKNOWN, description=access denied: channel [] creator org [Org2MSP], cause=null}
java.lang.Exception: io.grpc.StatusRuntimeException: UNKNOWN: access denied: channel [] creator org [Org2MSP]
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4140) ~[fabric-sdk-java-1.4.0.jar:?]
	at org.hyperledger.fabric.sdk.Channel.sendInstallProposal(Channel.java:2589) ~[fabric-sdk-java-1.4.0.jar:?]
	at org.hyperledger.fabric.sdk.HFClient.sendInstallProposal(HFClient.java:666) ~[fabric-sdk-java-1.4.0.jar:?]
	at com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:69) ~[classes/:?]
	at com.sansec.fabrictest.sdk.SdkMain.main(SdkMain.java:80) ~[classes/:?]
Caused by: io.grpc.StatusRuntimeException: UNKNOWN: access denied: channel [] creator org [Org2MSP]
	at io.grpc.Status.asRuntimeException(Status.java:530) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:482) ~[grpc-stub-1.17.1.jar:1.17.1]
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:694) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:397) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:459) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:63) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:546) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$600(ClientCallImpl.java:467) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:584) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123) ~[grpc-core-1.17.1.jar:1.17.1]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_241]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_241]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_241]
[2023-06-02 19:19:20] [ERROR] org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4140) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 3, name: } Sending proposal with transaction: 24888c3eec3a4692b4fe30461eb1cab9d507b2b61f0a90bc7c5f8e0310364b6e to Peer{ id: 2, name: peer1.org1.example.com, channelName: null, url: grpcs://peer1.org1.example.com:8051} failed because of: gRPC failure=Status{code=UNKNOWN, description=access denied: channel [] creator org [Org2MSP], cause=null}
java.lang.Exception: io.grpc.StatusRuntimeException: UNKNOWN: access denied: channel [] creator org [Org2MSP]
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4140) ~[fabric-sdk-java-1.4.0.jar:?]
	at org.hyperledger.fabric.sdk.Channel.sendInstallProposal(Channel.java:2589) ~[fabric-sdk-java-1.4.0.jar:?]
	at org.hyperledger.fabric.sdk.HFClient.sendInstallProposal(HFClient.java:666) ~[fabric-sdk-java-1.4.0.jar:?]
	at com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:69) ~[classes/:?]
	at com.sansec.fabrictest.sdk.SdkMain.main(SdkMain.java:80) ~[classes/:?]
Caused by: io.grpc.StatusRuntimeException: UNKNOWN: access denied: channel [] creator org [Org2MSP]
	at io.grpc.Status.asRuntimeException(Status.java:530) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:482) ~[grpc-stub-1.17.1.jar:1.17.1]
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:694) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:397) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:459) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:63) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:546) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$600(ClientCallImpl.java:467) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:584) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123) ~[grpc-core-1.17.1.jar:1.17.1]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_241]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_241]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_241]
[2023-06-02 19:19:20] [ERROR] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:74) com.sansec.fabrictest.sdk.FabricClient - Channel Channel{id: 3, name: } Sending proposal with transaction: 24888c3eec3a4692b4fe30461eb1cab9d507b2b61f0a90bc7c5f8e0310364b6e to Peer{ id: 1, name: peer0.org1.example.com, channelName: null, url: grpcs://peer0.org1.example.com:7051} failed because of: gRPC failure=Status{code=UNKNOWN, description=access denied: channel [] creator org [Org2MSP], cause=null} installed fail
[2023-06-02 19:19:20] [ERROR] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:74) com.sansec.fabrictest.sdk.FabricClient - Channel Channel{id: 3, name: } Sending proposal with transaction: 24888c3eec3a4692b4fe30461eb1cab9d507b2b61f0a90bc7c5f8e0310364b6e to Peer{ id: 2, name: peer1.org1.example.com, channelName: null, url: grpcs://peer1.org1.example.com:8051} failed because of: gRPC failure=Status{code=UNKNOWN, description=access denied: channel [] creator org [Org2MSP], cause=null} installed fail
[2023-06-02 19:19:25] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-02 19:19:26] [INFO ] org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder.createNetModeTransaction(InstallProposalBuilder.java:244) org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder - Installing 'basicinfo::basicinfo::1.0' language Go chaincode from directory: 'D:\SansecData5\chaincode\src\basicinfo' with source location: 'src\basicinfo'. chaincodePath:'basicinfo'
[2023-06-02 19:19:28] [ERROR] org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4140) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 3, name: } Sending proposal with transaction: 64c2fdee6b5f4bdee55d3da1a042d2d7fefead004bdfdb74ceef03c1e7d7bb31 to Peer{ id: 1, name: peer0.org1.example.com, channelName: null, url: grpcs://peer0.org1.example.com:7051} failed because of: gRPC failure=Status{code=UNKNOWN, description=access denied: channel [] creator org [Org2MSP], cause=null}
java.lang.Exception: io.grpc.StatusRuntimeException: UNKNOWN: access denied: channel [] creator org [Org2MSP]
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4140) ~[fabric-sdk-java-1.4.0.jar:?]
	at org.hyperledger.fabric.sdk.Channel.sendInstallProposal(Channel.java:2589) ~[fabric-sdk-java-1.4.0.jar:?]
	at org.hyperledger.fabric.sdk.HFClient.sendInstallProposal(HFClient.java:666) ~[fabric-sdk-java-1.4.0.jar:?]
	at com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:69) ~[classes/:?]
	at com.sansec.fabrictest.sdk.SdkMain.main(SdkMain.java:80) ~[classes/:?]
Caused by: io.grpc.StatusRuntimeException: UNKNOWN: access denied: channel [] creator org [Org2MSP]
	at io.grpc.Status.asRuntimeException(Status.java:530) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:482) ~[grpc-stub-1.17.1.jar:1.17.1]
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:694) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:397) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:459) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:63) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:546) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$600(ClientCallImpl.java:467) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:584) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123) ~[grpc-core-1.17.1.jar:1.17.1]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_241]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_241]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_241]
[2023-06-02 19:19:28] [ERROR] org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4140) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 3, name: } Sending proposal with transaction: 64c2fdee6b5f4bdee55d3da1a042d2d7fefead004bdfdb74ceef03c1e7d7bb31 to Peer{ id: 2, name: peer1.org1.example.com, channelName: null, url: grpcs://peer1.org1.example.com:8051} failed because of: gRPC failure=Status{code=UNKNOWN, description=access denied: channel [] creator org [Org2MSP], cause=null}
java.lang.Exception: io.grpc.StatusRuntimeException: UNKNOWN: access denied: channel [] creator org [Org2MSP]
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4140) ~[fabric-sdk-java-1.4.0.jar:?]
	at org.hyperledger.fabric.sdk.Channel.sendInstallProposal(Channel.java:2589) ~[fabric-sdk-java-1.4.0.jar:?]
	at org.hyperledger.fabric.sdk.HFClient.sendInstallProposal(HFClient.java:666) ~[fabric-sdk-java-1.4.0.jar:?]
	at com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:69) ~[classes/:?]
	at com.sansec.fabrictest.sdk.SdkMain.main(SdkMain.java:80) ~[classes/:?]
Caused by: io.grpc.StatusRuntimeException: UNKNOWN: access denied: channel [] creator org [Org2MSP]
	at io.grpc.Status.asRuntimeException(Status.java:530) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:482) ~[grpc-stub-1.17.1.jar:1.17.1]
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:694) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:397) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:459) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:63) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:546) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$600(ClientCallImpl.java:467) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:584) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123) ~[grpc-core-1.17.1.jar:1.17.1]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_241]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_241]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_241]
[2023-06-02 19:19:28] [ERROR] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:74) com.sansec.fabrictest.sdk.FabricClient - Channel Channel{id: 3, name: } Sending proposal with transaction: 64c2fdee6b5f4bdee55d3da1a042d2d7fefead004bdfdb74ceef03c1e7d7bb31 to Peer{ id: 1, name: peer0.org1.example.com, channelName: null, url: grpcs://peer0.org1.example.com:7051} failed because of: gRPC failure=Status{code=UNKNOWN, description=access denied: channel [] creator org [Org2MSP], cause=null} installed fail
[2023-06-02 19:19:28] [ERROR] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:74) com.sansec.fabrictest.sdk.FabricClient - Channel Channel{id: 3, name: } Sending proposal with transaction: 64c2fdee6b5f4bdee55d3da1a042d2d7fefead004bdfdb74ceef03c1e7d7bb31 to Peer{ id: 2, name: peer1.org1.example.com, channelName: null, url: grpcs://peer1.org1.example.com:8051} failed because of: gRPC failure=Status{code=UNKNOWN, description=access denied: channel [] creator org [Org2MSP], cause=null} installed fail
[2023-06-02 19:20:13] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-02 19:20:14] [INFO ] org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder.createNetModeTransaction(InstallProposalBuilder.java:244) org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder - Installing 'basicinfo::basicinfo::1.0' language Go chaincode from directory: 'D:\SansecData5\chaincode\src\basicinfo' with source location: 'src\basicinfo'. chaincodePath:'basicinfo'
[2023-06-02 19:20:16] [INFO ] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:72) com.sansec.fabrictest.sdk.FabricClient - peer0.org1.example.com installed sucess
[2023-06-02 19:20:16] [INFO ] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:72) com.sansec.fabrictest.sdk.FabricClient - peer1.org1.example.com installed sucess
[2023-06-14 17:19:52] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-14 17:19:52] [INFO ] org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder.createNetModeTransaction(InstallProposalBuilder.java:244) org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder - Installing 'basicinfo::basicinfo::1.0' language Go chaincode from directory: 'D:\SansecData5\chaincode\src\basicinfo' with source location: 'src\basicinfo'. chaincodePath:'basicinfo'
[2023-06-14 17:19:55] [INFO ] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:72) com.sansec.fabrictest.sdk.FabricClient - peer0.org1.example.com installed sucess
[2023-06-14 17:19:55] [INFO ] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:72) com.sansec.fabrictest.sdk.FabricClient - peer1.org1.example.com installed sucess
[2023-06-15 09:35:35] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 09:35:35] [INFO ] org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder.createNetModeTransaction(InstallProposalBuilder.java:244) org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder - Installing 'basicinfo::basicinfo::1.0' language Go chaincode from directory: 'D:\SansecData5\chaincode\src\basicinfo' with source location: 'src\basicinfo'. chaincodePath:'basicinfo'
[2023-06-15 09:35:37] [INFO ] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:76) com.sansec.fabrictest.sdk.FabricClient - peer0.org1.example.com installed sucess
[2023-06-15 09:35:37] [INFO ] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:76) com.sansec.fabrictest.sdk.FabricClient - peer1.org1.example.com installed sucess
[2023-06-15 09:45:33] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 09:45:35] [INFO ] org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1149) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 3, name: mychannel} eventThread started shutdown: false  thread: null 
[2023-06-15 09:45:51] [INFO ] com.sansec.fabrictest.sdk.FabricClient.initChaincode(FabricClient.java:113) com.sansec.fabrictest.sdk.FabricClient - peer0.org1.example.com init sucess
[2023-06-15 10:12:07] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 10:12:09] [INFO ] org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1149) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 3, name: mychannel} eventThread started shutdown: false  thread: null 
[2023-06-15 10:12:59] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 10:13:00] [INFO ] org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1149) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 3, name: mychannel} eventThread started shutdown: false  thread: null 
[2023-06-15 10:13:33] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 10:13:34] [INFO ] org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1149) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 3, name: mychannel} eventThread started shutdown: false  thread: null 
[2023-06-15 10:13:34] [ERROR] com.sansec.fabrictest.sdk.FabricClient.upgradeChaincode(FabricClient.java:160) com.sansec.fabrictest.sdk.FabricClient - cannot get package for chaincode (basicinfo:2.0) upgrade fail
[2023-06-15 10:16:24] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 10:16:24] [INFO ] org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder.createNetModeTransaction(InstallProposalBuilder.java:244) org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder - Installing 'basicinfo::basicinfo::1.0' language Go chaincode from directory: 'D:\SansecData5\chaincode\src\basicinfo' with source location: 'src\basicinfo'. chaincodePath:'basicinfo'
[2023-06-15 10:16:26] [ERROR] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:78) com.sansec.fabrictest.sdk.FabricClient - error installing chaincode code basicinfo:1.0(chaincode /var/hyperledger/production/chaincodes/basicinfo.1.0 exists) installed fail
[2023-06-15 10:16:26] [ERROR] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:78) com.sansec.fabrictest.sdk.FabricClient - error installing chaincode code basicinfo:1.0(chaincode /var/hyperledger/production/chaincodes/basicinfo.1.0 exists) installed fail
[2023-06-15 10:17:13] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 10:17:14] [INFO ] org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder.createNetModeTransaction(InstallProposalBuilder.java:244) org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder - Installing 'basicinfo::basicinfo::2.0' language Go chaincode from directory: 'D:\SansecData5\chaincode\src\basicinfo' with source location: 'src\basicinfo'. chaincodePath:'basicinfo'
[2023-06-15 10:17:16] [INFO ] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:76) com.sansec.fabrictest.sdk.FabricClient - peer0.org1.example.com installed sucess
[2023-06-15 10:17:16] [INFO ] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:76) com.sansec.fabrictest.sdk.FabricClient - peer1.org1.example.com installed sucess
[2023-06-15 10:17:35] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 10:17:36] [INFO ] org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1149) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 3, name: mychannel} eventThread started shutdown: false  thread: null 
[2023-06-15 10:17:52] [INFO ] com.sansec.fabrictest.sdk.FabricClient.upgradeChaincode(FabricClient.java:158) com.sansec.fabrictest.sdk.FabricClient - peer0.org1.example.com upgrade sucess
[2023-06-15 10:26:28] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 10:26:31] [INFO ] org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1149) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 4, name: mychannel} eventThread started shutdown: false  thread: null 
[2023-06-15 10:26:31] [INFO ] com.sansec.fabrictest.sdk.FabricClient.invoke(FabricClient.java:197) com.sansec.fabrictest.sdk.FabricClient - peer0.org1.example.com invoke proposal save sucess
[2023-06-15 10:26:31] [ERROR] com.sansec.fabrictest.sdk.FabricClient.invoke(FabricClient.java:200) com.sansec.fabrictest.sdk.FabricClient - cannot retrieve package for chaincode basicinfo/2.0, error open /var/hyperledger/production/chaincodes/basicinfo.2.0: no such file or directory invoke proposal save fail on peer0.org2.example.com
[2023-06-15 10:54:23] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 10:54:24] [INFO ] org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder.createNetModeTransaction(InstallProposalBuilder.java:244) org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder - Installing 'basicinfo::basicinfo::2.0' language Go chaincode from directory: 'D:\SansecData5\chaincode\src\basicinfo' with source location: 'src\basicinfo'. chaincodePath:'basicinfo'
[2023-06-15 10:54:26] [ERROR] org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4140) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 3, name: } Sending proposal with transaction: c6f5e0b3fd6536ff0dd9c0fcc3aa950477b9810c3785904e3d1bb50a22b1b8c6 to Peer{ id: 1, name: peer0.org1.example.com, channelName: null, url: grpcs://peer0.org1.example.com:7051} failed because of: gRPC failure=Status{code=UNAVAILABLE, description=io exception, cause=javax.net.ssl.SSLHandshakeException: General OpenSslEngine problem
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.handshakeException(ReferenceCountedOpenSslEngine.java:1929)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.wrap(ReferenceCountedOpenSslEngine.java:836)
	at javax.net.ssl.SSLEngine.wrap(SSLEngine.java:509)
	at io.netty.handler.ssl.SslHandler.wrap(SslHandler.java:1042)
	at io.netty.handler.ssl.SslHandler.wrapNonAppData(SslHandler.java:928)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1419)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1246)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1295)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:529)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:468)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:450)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:317)
	at sun.security.validator.Validator.validate(Validator.java:262)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:330)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:279)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:144)
	at io.netty.handler.ssl.ReferenceCountedOpenSslClientContext$ExtendedTrustManagerVerifyCallback.verify(ReferenceCountedOpenSslClientContext.java:234)
	at io.netty.handler.ssl.ReferenceCountedOpenSslContext$AbstractCertificateVerifier.verify(ReferenceCountedOpenSslContext.java:785)
	at io.netty.internal.tcnative.CertificateVerifierTask.runTask(CertificateVerifierTask.java:36)
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:48)
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:42)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.runAndResetNeedTask(ReferenceCountedOpenSslEngine.java:1518)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.access$700(ReferenceCountedOpenSslEngine.java:94)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine$TaskDecorator.run(ReferenceCountedOpenSslEngine.java:1493)
	at io.netty.handler.ssl.SslHandler.runDelegatedTasks(SslHandler.java:1559)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1405)
	... 21 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:445)
	... 36 more
}
java.lang.Exception: io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4140) ~[fabric-sdk-java-1.4.0.jar:?]
	at org.hyperledger.fabric.sdk.Channel.sendInstallProposal(Channel.java:2589) ~[fabric-sdk-java-1.4.0.jar:?]
	at org.hyperledger.fabric.sdk.HFClient.sendInstallProposal(HFClient.java:666) ~[fabric-sdk-java-1.4.0.jar:?]
	at com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:73) ~[classes/:?]
	at com.sansec.fabrictest.sdk.SdkMain.installChaincodeOrg2(SdkMain.java:129) ~[classes/:?]
	at com.sansec.fabrictest.sdk.SdkMain.main(SdkMain.java:53) ~[classes/:?]
Caused by: io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
	at io.grpc.Status.asRuntimeException(Status.java:530) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:482) ~[grpc-stub-1.17.1.jar:1.17.1]
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:694) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:397) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:459) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:63) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:546) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$600(ClientCallImpl.java:467) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:584) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123) ~[grpc-core-1.17.1.jar:1.17.1]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_241]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_241]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_241]
Caused by: javax.net.ssl.SSLHandshakeException: General OpenSslEngine problem
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.handshakeException(ReferenceCountedOpenSslEngine.java:1929) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.wrap(ReferenceCountedOpenSslEngine.java:836) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at javax.net.ssl.SSLEngine.wrap(SSLEngine.java:509) ~[?:1.8.0_241]
	at io.netty.handler.ssl.SslHandler.wrap(SslHandler.java:1042) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.wrapNonAppData(SslHandler.java:928) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1419) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1246) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1295) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:529) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:468) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_241]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:450) ~[?:1.8.0_241]
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:317) ~[?:1.8.0_241]
	at sun.security.validator.Validator.validate(Validator.java:262) ~[?:1.8.0_241]
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:330) ~[?:1.8.0_241]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:279) ~[?:1.8.0_241]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:144) ~[?:1.8.0_241]
	at io.netty.handler.ssl.ReferenceCountedOpenSslClientContext$ExtendedTrustManagerVerifyCallback.verify(ReferenceCountedOpenSslClientContext.java:234) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslContext$AbstractCertificateVerifier.verify(ReferenceCountedOpenSslContext.java:785) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.internal.tcnative.CertificateVerifierTask.runTask(CertificateVerifierTask.java:36) ~[netty-tcnative-classes-2.0.60.Final.jar:2.0.60.Final]
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:48) ~[netty-tcnative-classes-2.0.60.Final.jar:2.0.60.Final]
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:42) ~[netty-tcnative-classes-2.0.60.Final.jar:2.0.60.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.runAndResetNeedTask(ReferenceCountedOpenSslEngine.java:1518) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.access$700(ReferenceCountedOpenSslEngine.java:94) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine$TaskDecorator.run(ReferenceCountedOpenSslEngine.java:1493) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.runDelegatedTasks(SslHandler.java:1559) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1405) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1246) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1295) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:529) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:468) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_241]
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141) ~[?:1.8.0_241]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126) ~[?:1.8.0_241]
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280) ~[?:1.8.0_241]
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:445) ~[?:1.8.0_241]
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:317) ~[?:1.8.0_241]
	at sun.security.validator.Validator.validate(Validator.java:262) ~[?:1.8.0_241]
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:330) ~[?:1.8.0_241]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:279) ~[?:1.8.0_241]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:144) ~[?:1.8.0_241]
	at io.netty.handler.ssl.ReferenceCountedOpenSslClientContext$ExtendedTrustManagerVerifyCallback.verify(ReferenceCountedOpenSslClientContext.java:234) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslContext$AbstractCertificateVerifier.verify(ReferenceCountedOpenSslContext.java:785) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.internal.tcnative.CertificateVerifierTask.runTask(CertificateVerifierTask.java:36) ~[netty-tcnative-classes-2.0.60.Final.jar:2.0.60.Final]
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:48) ~[netty-tcnative-classes-2.0.60.Final.jar:2.0.60.Final]
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:42) ~[netty-tcnative-classes-2.0.60.Final.jar:2.0.60.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.runAndResetNeedTask(ReferenceCountedOpenSslEngine.java:1518) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.access$700(ReferenceCountedOpenSslEngine.java:94) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine$TaskDecorator.run(ReferenceCountedOpenSslEngine.java:1493) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.runDelegatedTasks(SslHandler.java:1559) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1405) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1246) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1295) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:529) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:468) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_241]
[2023-06-15 10:54:26] [ERROR] org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4140) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 3, name: } Sending proposal with transaction: c6f5e0b3fd6536ff0dd9c0fcc3aa950477b9810c3785904e3d1bb50a22b1b8c6 to Peer{ id: 2, name: peer1.org1.example.com, channelName: null, url: grpcs://peer1.org1.example.com:8051} failed because of: gRPC failure=Status{code=UNAVAILABLE, description=io exception, cause=javax.net.ssl.SSLHandshakeException: General OpenSslEngine problem
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.handshakeException(ReferenceCountedOpenSslEngine.java:1929)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.wrap(ReferenceCountedOpenSslEngine.java:836)
	at javax.net.ssl.SSLEngine.wrap(SSLEngine.java:509)
	at io.netty.handler.ssl.SslHandler.wrap(SslHandler.java:1042)
	at io.netty.handler.ssl.SslHandler.wrapNonAppData(SslHandler.java:928)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1419)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1246)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1295)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:529)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:468)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:450)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:317)
	at sun.security.validator.Validator.validate(Validator.java:262)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:330)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:279)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:144)
	at io.netty.handler.ssl.ReferenceCountedOpenSslClientContext$ExtendedTrustManagerVerifyCallback.verify(ReferenceCountedOpenSslClientContext.java:234)
	at io.netty.handler.ssl.ReferenceCountedOpenSslContext$AbstractCertificateVerifier.verify(ReferenceCountedOpenSslContext.java:785)
	at io.netty.internal.tcnative.CertificateVerifierTask.runTask(CertificateVerifierTask.java:36)
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:48)
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:42)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.runAndResetNeedTask(ReferenceCountedOpenSslEngine.java:1518)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.access$700(ReferenceCountedOpenSslEngine.java:94)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine$TaskDecorator.run(ReferenceCountedOpenSslEngine.java:1493)
	at io.netty.handler.ssl.SslHandler.runDelegatedTasks(SslHandler.java:1559)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1405)
	... 21 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:445)
	... 36 more
}
java.lang.Exception: io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4140) ~[fabric-sdk-java-1.4.0.jar:?]
	at org.hyperledger.fabric.sdk.Channel.sendInstallProposal(Channel.java:2589) ~[fabric-sdk-java-1.4.0.jar:?]
	at org.hyperledger.fabric.sdk.HFClient.sendInstallProposal(HFClient.java:666) ~[fabric-sdk-java-1.4.0.jar:?]
	at com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:73) ~[classes/:?]
	at com.sansec.fabrictest.sdk.SdkMain.installChaincodeOrg2(SdkMain.java:129) ~[classes/:?]
	at com.sansec.fabrictest.sdk.SdkMain.main(SdkMain.java:53) ~[classes/:?]
Caused by: io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
	at io.grpc.Status.asRuntimeException(Status.java:530) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:482) ~[grpc-stub-1.17.1.jar:1.17.1]
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:694) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:397) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:459) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:63) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:546) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$600(ClientCallImpl.java:467) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:584) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37) ~[grpc-core-1.17.1.jar:1.17.1]
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123) ~[grpc-core-1.17.1.jar:1.17.1]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_241]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_241]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_241]
Caused by: javax.net.ssl.SSLHandshakeException: General OpenSslEngine problem
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.handshakeException(ReferenceCountedOpenSslEngine.java:1929) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.wrap(ReferenceCountedOpenSslEngine.java:836) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at javax.net.ssl.SSLEngine.wrap(SSLEngine.java:509) ~[?:1.8.0_241]
	at io.netty.handler.ssl.SslHandler.wrap(SslHandler.java:1042) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.wrapNonAppData(SslHandler.java:928) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1419) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1246) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1295) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:529) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:468) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_241]
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:450) ~[?:1.8.0_241]
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:317) ~[?:1.8.0_241]
	at sun.security.validator.Validator.validate(Validator.java:262) ~[?:1.8.0_241]
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:330) ~[?:1.8.0_241]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:279) ~[?:1.8.0_241]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:144) ~[?:1.8.0_241]
	at io.netty.handler.ssl.ReferenceCountedOpenSslClientContext$ExtendedTrustManagerVerifyCallback.verify(ReferenceCountedOpenSslClientContext.java:234) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslContext$AbstractCertificateVerifier.verify(ReferenceCountedOpenSslContext.java:785) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.internal.tcnative.CertificateVerifierTask.runTask(CertificateVerifierTask.java:36) ~[netty-tcnative-classes-2.0.60.Final.jar:2.0.60.Final]
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:48) ~[netty-tcnative-classes-2.0.60.Final.jar:2.0.60.Final]
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:42) ~[netty-tcnative-classes-2.0.60.Final.jar:2.0.60.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.runAndResetNeedTask(ReferenceCountedOpenSslEngine.java:1518) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.access$700(ReferenceCountedOpenSslEngine.java:94) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine$TaskDecorator.run(ReferenceCountedOpenSslEngine.java:1493) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.runDelegatedTasks(SslHandler.java:1559) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1405) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1246) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1295) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:529) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:468) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_241]
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141) ~[?:1.8.0_241]
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126) ~[?:1.8.0_241]
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280) ~[?:1.8.0_241]
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:445) ~[?:1.8.0_241]
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:317) ~[?:1.8.0_241]
	at sun.security.validator.Validator.validate(Validator.java:262) ~[?:1.8.0_241]
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:330) ~[?:1.8.0_241]
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:279) ~[?:1.8.0_241]
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:144) ~[?:1.8.0_241]
	at io.netty.handler.ssl.ReferenceCountedOpenSslClientContext$ExtendedTrustManagerVerifyCallback.verify(ReferenceCountedOpenSslClientContext.java:234) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslContext$AbstractCertificateVerifier.verify(ReferenceCountedOpenSslContext.java:785) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.internal.tcnative.CertificateVerifierTask.runTask(CertificateVerifierTask.java:36) ~[netty-tcnative-classes-2.0.60.Final.jar:2.0.60.Final]
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:48) ~[netty-tcnative-classes-2.0.60.Final.jar:2.0.60.Final]
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:42) ~[netty-tcnative-classes-2.0.60.Final.jar:2.0.60.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.runAndResetNeedTask(ReferenceCountedOpenSslEngine.java:1518) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.access$700(ReferenceCountedOpenSslEngine.java:94) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine$TaskDecorator.run(ReferenceCountedOpenSslEngine.java:1493) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.runDelegatedTasks(SslHandler.java:1559) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1405) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1246) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1295) ~[netty-handler-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:529) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:468) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290) ~[netty-codec-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562) ~[netty-transport-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.92.Final.jar:4.1.92.Final]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_241]
[2023-06-15 10:54:26] [ERROR] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:78) com.sansec.fabrictest.sdk.FabricClient - Channel Channel{id: 3, name: } Sending proposal with transaction: c6f5e0b3fd6536ff0dd9c0fcc3aa950477b9810c3785904e3d1bb50a22b1b8c6 to Peer{ id: 1, name: peer0.org1.example.com, channelName: null, url: grpcs://peer0.org1.example.com:7051} failed because of: gRPC failure=Status{code=UNAVAILABLE, description=io exception, cause=javax.net.ssl.SSLHandshakeException: General OpenSslEngine problem
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.handshakeException(ReferenceCountedOpenSslEngine.java:1929)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.wrap(ReferenceCountedOpenSslEngine.java:836)
	at javax.net.ssl.SSLEngine.wrap(SSLEngine.java:509)
	at io.netty.handler.ssl.SslHandler.wrap(SslHandler.java:1042)
	at io.netty.handler.ssl.SslHandler.wrapNonAppData(SslHandler.java:928)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1419)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1246)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1295)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:529)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:468)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:450)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:317)
	at sun.security.validator.Validator.validate(Validator.java:262)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:330)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:279)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:144)
	at io.netty.handler.ssl.ReferenceCountedOpenSslClientContext$ExtendedTrustManagerVerifyCallback.verify(ReferenceCountedOpenSslClientContext.java:234)
	at io.netty.handler.ssl.ReferenceCountedOpenSslContext$AbstractCertificateVerifier.verify(ReferenceCountedOpenSslContext.java:785)
	at io.netty.internal.tcnative.CertificateVerifierTask.runTask(CertificateVerifierTask.java:36)
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:48)
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:42)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.runAndResetNeedTask(ReferenceCountedOpenSslEngine.java:1518)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.access$700(ReferenceCountedOpenSslEngine.java:94)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine$TaskDecorator.run(ReferenceCountedOpenSslEngine.java:1493)
	at io.netty.handler.ssl.SslHandler.runDelegatedTasks(SslHandler.java:1559)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1405)
	... 21 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:445)
	... 36 more
} installed fail
[2023-06-15 10:54:26] [ERROR] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:78) com.sansec.fabrictest.sdk.FabricClient - Channel Channel{id: 3, name: } Sending proposal with transaction: c6f5e0b3fd6536ff0dd9c0fcc3aa950477b9810c3785904e3d1bb50a22b1b8c6 to Peer{ id: 2, name: peer1.org1.example.com, channelName: null, url: grpcs://peer1.org1.example.com:8051} failed because of: gRPC failure=Status{code=UNAVAILABLE, description=io exception, cause=javax.net.ssl.SSLHandshakeException: General OpenSslEngine problem
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.handshakeException(ReferenceCountedOpenSslEngine.java:1929)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.wrap(ReferenceCountedOpenSslEngine.java:836)
	at javax.net.ssl.SSLEngine.wrap(SSLEngine.java:509)
	at io.netty.handler.ssl.SslHandler.wrap(SslHandler.java:1042)
	at io.netty.handler.ssl.SslHandler.wrapNonAppData(SslHandler.java:928)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1419)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1246)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1295)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:529)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:468)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:450)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:317)
	at sun.security.validator.Validator.validate(Validator.java:262)
	at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:330)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:279)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:144)
	at io.netty.handler.ssl.ReferenceCountedOpenSslClientContext$ExtendedTrustManagerVerifyCallback.verify(ReferenceCountedOpenSslClientContext.java:234)
	at io.netty.handler.ssl.ReferenceCountedOpenSslContext$AbstractCertificateVerifier.verify(ReferenceCountedOpenSslContext.java:785)
	at io.netty.internal.tcnative.CertificateVerifierTask.runTask(CertificateVerifierTask.java:36)
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:48)
	at io.netty.internal.tcnative.SSLTask.run(SSLTask.java:42)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.runAndResetNeedTask(ReferenceCountedOpenSslEngine.java:1518)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine.access$700(ReferenceCountedOpenSslEngine.java:94)
	at io.netty.handler.ssl.ReferenceCountedOpenSslEngine$TaskDecorator.run(ReferenceCountedOpenSslEngine.java:1493)
	at io.netty.handler.ssl.SslHandler.runDelegatedTasks(SslHandler.java:1559)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1405)
	... 21 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(SunCertPathBuilder.java:141)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(SunCertPathBuilder.java:126)
	at java.security.cert.CertPathBuilder.build(CertPathBuilder.java:280)
	at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:445)
	... 36 more
} installed fail
[2023-06-15 10:55:46] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 10:55:47] [INFO ] org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder.createNetModeTransaction(InstallProposalBuilder.java:244) org.hyperledger.fabric.sdk.transaction.InstallProposalBuilder - Installing 'basicinfo::basicinfo::2.0' language Go chaincode from directory: 'D:\SansecData5\chaincode\src\basicinfo' with source location: 'src\basicinfo'. chaincodePath:'basicinfo'
[2023-06-15 10:55:47] [INFO ] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:76) com.sansec.fabrictest.sdk.FabricClient - peer0.org2.example.com installed sucess
[2023-06-15 10:55:47] [INFO ] com.sansec.fabrictest.sdk.FabricClient.installChaincode(FabricClient.java:76) com.sansec.fabrictest.sdk.FabricClient - peer1.org2.example.com installed sucess
[2023-06-15 10:56:52] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 10:56:54] [INFO ] org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1149) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 4, name: mychannel} eventThread started shutdown: false  thread: null 
[2023-06-15 10:57:12] [INFO ] com.sansec.fabrictest.sdk.FabricClient.invoke(FabricClient.java:197) com.sansec.fabrictest.sdk.FabricClient - peer0.org1.example.com invoke proposal save sucess
[2023-06-15 10:57:12] [INFO ] com.sansec.fabrictest.sdk.FabricClient.invoke(FabricClient.java:197) com.sansec.fabrictest.sdk.FabricClient - peer0.org2.example.com invoke proposal save sucess
[2023-06-15 11:00:34] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 11:00:35] [INFO ] org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1149) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 3, name: mychannel} eventThread started shutdown: false  thread: null 
[2023-06-15 11:00:35] [INFO ] com.sansec.fabrictest.sdk.FabricClient.queryChaincode(FabricClient.java:237) com.sansec.fabrictest.sdk.FabricClient - data is <ByteString@51f49060 size=62>
[2023-06-15 18:23:50] [WARN ] org.hyperledger.fabric.sdk.helper.Config.<init>(Config.java:127) org.hyperledger.fabric.sdk.helper.Config - Failed to load any configuration from: config.properties. Using toolkit defaults
[2023-06-15 18:23:54] [INFO ] org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1149) org.hyperledger.fabric.sdk.Channel - Channel Channel{id: 3, name: mychannel} eventThread started shutdown: false  thread: null 
[2023-06-15 18:23:54] [ERROR] com.sansec.fabrictest.sdk.FabricClient.queryChaincode(FabricClient.java:241) com.sansec.fabrictest.sdk.FabricClient - data get error make sure the chaincode basicinfo has been successfully instantiated and try again: chaincode basicinfo not found
